src/app/public/modules/datepicker/datepicker-input.directive.ts

```diff
@@ -245,6 +245,7 @@ export class SkyDatepickerInputDirective
       .distinctUntilChanged()
       .takeUntil(this.ngUnsubscribe)
       .subscribe((value: Date) => {
+        this.isFirstChange = false;
```

Comment:
Just to double check - are we guaranteed that anything coming in from the observable here isn't the first change? Because changing this before we set the value in the next line will negate all the checks we have in the `value` setter for the first change.

Reply:
This observable comes from the user selecting a date from the calendar drop down, so it should always be considered a "real" change, and not an initial change.